base/deprecated.jl

```diff
@@ -2772,6 +2772,8 @@ end
 
 @deprecate findin(a, b) find(occursin(b), a)
 
+@deprecate findn(x::AbstractArray) (I = find(!iszero, x); (getindex.(I, 1), getindex.(I, 2)))
```

Comment:
Is it worth trying to support non-2d arrays?

Reply:
Good catch. I've added a method for vectors and another for general arrays, and restricted this one to matrices.